CONTRIBUTE.md

```diff
@@ -153,3 +153,18 @@ let sol = decomp
      .solve_triangular(UPLO::Lower, Diag::NonUnit, &Array::eye(n_features))?
      .without_lapack();
 ```
+
+## Benchmarking
+
+It is important to the project that we have benchmarks in place to evaluate the benefit of performance related changes. To make that process easier we provide some guidelines for writing benchmarks.
+
+1. Test for a variety of sample sizes for most algorithms [1_000, 10_000, 20_000] will be sufficient
+2. Test of variety feature dimensions
```

Comment:
"Test for a variety of feature dimensions. Two is usually enough for most algorithms. The following are suggested feature dimensions to use for different types of algorithms:"